lms/services/course.py

```diff
@@ -75,7 +75,7 @@ def get(self, authority_provided_id):
 
     def _create_legacy(self, authority_provided_id):
         course = LegacyCourse(
-            consumer_key=self._application_instance.consumer_key,
```

Comment:
This could stay and will'll just populate the value for LTI1.1 if needed for backwards compatibility from outside LMS (analytics? metabase?).